tendrl/ceph_integration/objects/pool/atoms/delete/__init__.py

```diff
@@ -50,10 +50,13 @@ def run(self):
             )
             return False
 
-        tendrl_ns.ceph_integration.objects.Pool(
-            pool_id=pool_id,
-            deleted="True"
-        ).save()
+        tendrl_ns.etcd_orm.client.delete(
```

Comment:
add a TODO to make this change via the namespace tree and not etcd orm for laters